alf/algorithms/config.py

```diff
@@ -159,6 +160,11 @@ def __init__(self,
                 states will be taken from the replay buffer; otherwise they will
                 be set to 0. In the case of True, the ``train_state_spec`` of an
                 algorithm should always be a subset of the ``rollout_state_spec``.
+            mask_out_loss_for_last_step (bool): If True, the loss for the last
+                step of each sequence will be masked out. For RL training,
```

Comment:
``mask_out_loss_for_last_step`` is True does not always do loss masking. The current logic also requires  ``is_rl()`` is True for ``mask_out_loss_for_last_step`` to take effect.

Reply:
If not is_rl(), then there is no last step either.